core/src/apps/ethereum/layout.py

```
@@ -332,10 +332,10 @@
     type_name = get_type_name(field)
 
     if array_index is not None:
-        title = limit_str(".".join(parent_objects + [name]))
+        title = ".".join(parent_objects + [name])
```

Ruff (0.8.2)

Consider `[*parent_objects, name]` instead of concatenation

Replace with `[*parent_objects, name]`

(RUF005)